server/secrets/README.md

```diff
@@ -0,0 +1,6 @@
+# Otrego Secrets
+
+This directory contains the secret-logic for Otrego.
+
+
+* `testing/` -- keys / certs for running locally. **never** use in production.
```

Comment:
Could you mention:
  a) Sample of how you might use these, and
  b) If they're expected to ever need to be recreated, and how to generate updated equivalents if so?